views/includes/save-and-continue-button.html

```diff
@@ -5,9 +5,7 @@
     {% set textContinueButton = "Save and continue" %}
   {% endif %}
 {% else %}
-  {% if OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 === "true" %}
-    {% set textContinueButton = "Save and continue" %}
-  {% endif %}
+  {% set textContinueButton = "Save and continue" %}
```

Comment:
I think this logic could be simplified now (to set "Save and continue" to start with and have one conditional check that sets "Continue" if appropriate but not really a problem as this will all disappear when the next feature flag is removed.